tinygrad/lazy.py

```diff
@@ -111,13 +111,13 @@ def realize(self:LazyBuffer) -> LazyBuffer:
       # get real ops first
       if self.optype is BinaryOps: self.op = _ast_binaryops(self)
       elif self.optype is ReduceOps: self.op = _ast_reduceops(self)
-      elif self.optype is LoadOps: LOAD_OPS_DISPATCHER[cast(LoadOps, self.op.op)](self)
```

Comment:
This is a different statement. If you want changes like this, maybe we should remove optype entirely.